scripts/helm_deploy.sh

```diff
@@ -21,4 +21,8 @@ else
 fi
 
 cd helm/
+cd elife-xpub
+sudo -u elife -H helm init --client-only
+sudo -u elife -H helm dependency update .
+cd -
```

Comment:
personall I prefer `pushd / popd`

Reply:
I think the ideal would be a subshell that isolates from changing the working directory of the main script... Arcane but this will be pulled out of the project when stable